graylog2-server/src/main/java/org/graylog2/alerts/types/FieldContentValueAlertCondition.java

```diff
@@ -124,4 +126,9 @@ protected CheckResult runCheck() {
     public String getDescription() {
         return "field: " + field + ", value: " + value;
     }
+
+    @Override
+    public List<Message> getSearchHits() {
+        return searchHits;
```

Comment:
I'm not sure if this class is ever used concurrently from multiple threads, but if it was, this should either return a copy of the list (`ImmutableList#copyOf()`) or change the `searchHits` field to a `CopyOnWriteArrayList`.